test/e2e/memcached_test.go

```diff
@@ -132,6 +165,14 @@ func TestMemcached(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	defer func() {
```

Comment:
probably no need since deleting namespace also deletes the deployment inside it.